hack/setup_db.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -euo pipefail
+set -o errexit
+
+mkdir -p /tmp/postgres/data
+mkdir -p /tmp/postgres/sockets
+
+/usr/bin/initdb -D /tmp/postgres/data -U postgres
```

Comment:
Wont this fail when we're in an environment where the container is running as root?

Reply:
Yes, but what will be that use case? In CI, it will be run as a nonroot user, no?

Reply:
> Wont this fail when we're in an environment where the container is running as root?

What container? Why would this fail when it's running as root?

Reply:
this file will be used only from ci-unit-test. In earlier PR, the similar file was getting used from `unit-test` target and then the jenkins PR check was failing but now that we haven't changed any internal functionality for the `unit-test` target, it should not be an issue.

Reply:
> Why would this fail when it's running as root?

IIRC postgres itself has Opinions about what user owns a data directory.